lang/test/org/partiql/lang/eval/CastTestBase.kt

```diff
@@ -56,6 +56,46 @@ data class FixSemantics(val expectedQuality: CastQuality) : CastQualityStatus()
 
 abstract class CastTestBase : EvaluatorTestBase() {
 
+    fun ConfiguredCastCase.assertCase(
```

Comment:
Needed to move this here so we could access the `runEvaluator[Error]TestCase` functions of the new `EvaluatorTestBase`.